java/src/jmri/jmrix/ecos/swing/locodatabase/EcosLocoTableAction.java

```diff
@@ -110,7 +110,7 @@ protected EcosLocoAddress getByEcosObject(String object) {
 
     @Override
     protected void createModel() {
-        m = new BeanTableDataModel() {
+        m = new BeanTableDataModel<NamedBean>() { // this is a hack to get this to compile, as EcosLocoAddress is itself a NamedBean
```

Comment:
Is this comment correct? It seems the hack would not be needed if EcosLocoAddress is a NamedBean. (I think you meant the opposite of what you wrote in the comment, but am not sure.)

Reply:
Thanks for catching that.  There's a "not" missing in the comment:  EcosLocoAddress isn't a named bean, and it shouldn't be.  In the long run, this code needs to get redone to use its own Table instead of the beantable ones in an ad-hoc way.